juniper_official/linecard/check-fpc-cpu-memory-state-temp.rule

```diff
@@ -356,16 +413,16 @@ healthbot {
                 description "Sets health based on FPC Temperature";
                 frequency 2offset;
                 /*
-                 * Sets color to green when temperature is less-than-or-equal-to temp-low-threshold.
+                 * Sets color to red when temperature is-greater-than-or-equal-to temp-high-threshold .
                  */				
-                term is-temperature-normal {
```

Comment:
Retain normal check first for the triggers which are already adapted to this way

Reply:
Have changed the sequence to original format